docs/docs/development/new-shield.md

```diff
@@ -395,6 +411,30 @@ siblings:
 
 You should place a properly named `foo.zmk.yml` file in the directory next to your other shield values, and fill it out completely and accurately. See [Hardware Metadata Files](/docs/development/hardware-metadata-files) for the full details.
 
+## Build File
+
+To help you test/verify your firmware, update the `build.yaml` to list your particular board/shield combinations you want built whenever changes are published to GitHub. Open `build.yaml` with your editor and add a combination, e.g.:
+
+```
+# This file generates the GitHub Actions matrix
+# For simple board + shield combinations, add them
+# to the top level board and shield arrays, for more
+# control, add individual board + shield combinations to
+# the `include` property, e.g:
+#
+# board: [ "nice_nano_v2" ]
+# shield: [ "corne_left", "corne_right" ]
+# include:
+#   - board: bdn9_rev2
+#   - board: nice_nano_v2
+#     shield: reviung41
+#
+---
+include:
+  - board: nice_nano_v2
+    shield: <my_shield>
+```
+
```

Comment:
How about adding this paragraph here? I've seen this be a point of confusion a couple times. (couldn't put it in a suggestion with the code block.)

For split keyboards, you will need to specify the halves/siblings separately:
```yaml
[...]
include:
  - board: nice_nano_v2
    shield: <my_shield>_left
  - board: nice_nano_v2
    shield: <my_shield>_right
```

Reply:
Added.